### PR TITLE
Added Try what you'll build section

### PR DESCRIPTION
**Pull request by justineechen**

### CODE REVIEW

**Comment by justineechen** [comment]
Addressing https://github.com/OpenLiberty/guides-common/issues/244

File: README.adoc [hunk 1]
```diff
@@ -54,6 +54,34 @@ return a `200 OK` response that contains all of the JVM's system properties.
...
 include::{common-includes}/gitclone.adoc[]
 
+=== Try what you'll build
+To try out the application, first navigate to the `finish` directory and then run the following Maven goals to build the application and run it inside Open Liberty:
+[role='command']
+```
+mvn install
+```
+
+This command builds the application and creates a `.war` file in the target directory. It also
```
Comment (evelinec) [inline README.adoc]: This paragraph is not need for "try what you'll build".

File: README.adoc [hunk 2]
```diff
@@ -54,6 +54,34 @@ return a `200 OK` response that contains all of the JVM's system properties.
...
+configures and installs Open Liberty into the `target/liberty/wlp` directory.
+
+Next, run the Maven `liberty:start-server` goal:
+[role='command']
+```
+mvn liberty:start-server
+```
+
+This goal starts an Open Liberty server instance. Your Maven `pom.xml` is already configured to start
```
Comment (evelinec) [inline README.adoc]: This paragraph is not needed for "try what you'll build".

File: README.adoc [hunk 3]
```diff
@@ -54,6 +54,34 @@ return a `200 OK` response that contains all of the JVM's system properties.
 [role="command"]
 include::{common-includes}/gitclone.adoc[]
 
+=== Try what you'll build
+To try out the application, first navigate to the `finish` directory and then run the following Maven goals to build the application and run it inside Open Liberty:
```
Comment (evelinec) [inline README.adoc]: Use common adoc where applicable.

File: README.adoc [hunk 4]
```diff
@@ -54,6 +54,34 @@ return a `200 OK` response that contains all of the JVM's system properties.
...
+[role='command']
+```
+mvn liberty:start-server
+```
+
+This goal starts an Open Liberty server instance. Your Maven `pom.xml` is already configured to start
+the application in this server instance.
+
+Check out the service at the
+http://localhost:9080/LibertyProject/System/properties[http://localhost:9080/LibertyProject/System/properties^] URL.
```
Comment (evelinec) [inline README.adoc]: I think this syntax works `http://localhost:9080/LibertyProject/System/properties[^]`.

File: README.adoc (hunk 2 above)
Comment (justineechen) [reply README.adoc]: I think because this is another intro guides so we decided to keep the details in the guide.

File: README.adoc (hunk 3 above)
Comment (justineechen) [reply README.adoc]: We're keeping the commands separate for this intro guide so this one isn't applicable.

**Pull request merged**